src/app/templates/app/convert.html

```diff
@@ -20,30 +20,25 @@
 		&nbsp; From &nbsp;
 			 <select name="from_format" id="from_format">
 			 	  <option value="0" name ="here" selected="">-</option>
-				  <option value="Tag">Tag</option>
-				  <option value="RDF">RDF</option>
-				  <option value="Spreadsheet">Spreadsheet</option>
+				  <option value="TAG">Tag</option>
+				  <option value="RDFXML">RDF</option>
```

Comment:
Would we like to change the option text to RDFXML?
<img width="908" alt="Screenshot 2020-10-12 at 10 32 40 PM" src="https://user-images.githubusercontent.com/17159160/95772778-e9368400-0cda-11eb-91c7-70ae4f6cd280.png">

It shows `Upload SPDX RDFXML Document` in the upload box.

Reply:
Good suggestion - fixed in commit 981e0d5